truffle/contracts/Splitter.sol

```diff
@@ -0,0 +1,43 @@
+pragma solidity 0.5.10;
+
+import './Stoppable.sol';
+
+contract Splitter is Stoppable {
+    
+    struct Account {
+        uint    pendingWithdrawals;
+    }
+    
+    mapping(address => Account) accountStorage;
+
+    event LogEthSent(address sender, uint value, address receiver1, address receiver2);
+    event LogWithdrawed(address sender, uint amount);
+    
+    constructor() public payable {}
+    
+    function splitEth(address receiver1, address receiver2) public _onlyIfRunning payable returns(bool) {
+        require(msg.value <= msg.sender.balance, 'The sender balance shoud be higher the ether\'s value');
+
+        accountStorage[receiver1].pendingWithdrawals += msg.value / 2;
+        accountStorage[receiver2].pendingWithdrawals += msg.value / 2;
+
```

Comment:
A couple of observations with the previous two lines. 

1. It _may_ be cheaper to do `uint half = msg.value / 2;` one time and then use `half` when you need it. This would avoid doing the division OP twice, but it introduces a MSTORE, MREAD for the new memory variable, so it isn't all savings. 
2. There is no catch for an odd number. What if `msg.value` isn't an even number?

Reply:
This rounding error concern is important for the safe handling of funds. It's bad form to allow "dust" to exist so you need to detect the odd number and decide what to do with the remainder.